### ISSUE TITLE
gonic NixOS module

### ISSUE THREAD

**Issue by fgaz** [NixOS/nixpkgs#181965]
**Project description**
music streaming server / subsonic server API implementation

**Metadata**

* homepage URL: https://github.com/sentriz/gonic
* source URL: https://github.com/sentriz/gonic
* license: gpl3
* platforms: linux
* package: https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/gonic/default.nix

The package is already there, maintained by @Profpatsch. It'd be nice to have a module too!

**Comment by urandom2** [comment]
#96569

**Comment by autrimpo** [comment]
I can submit a module PR once #204476 is merged, as it adds some new options (filesystem watcher).
